### DISCUSSION TITLE
Not Location Fined

### DISCUSSION THREAD

**Discussion by Kawishraza** [community/community#28725] [GitHub Education]
i am applying for GitHub Student Developer Pack but my university not showing in search box, kindly help what I can do now ??
![Screenshot (76)](https://user-images.githubusercontent.com/110709174/183355308-1c861521-a625-4b10-9ecf-e6d50430d300.png)

**Comment by pasindu651** [comment]
Hi @Kawishraza, I suggest that you contact your university to issue an institutional email and/or open a support ticket about the issue and send Github your screenshot [here](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket).
Hope this helps!
